src/transformers/generation/logits_process.py

```diff
@@ -154,31 +154,24 @@ class MinNewTokensLengthLogitsProcessor(LogitsProcessor):
     >>> tokenizer = AutoTokenizer.from_pretrained("distilgpt2")
     >>> model = AutoModelForCausalLM.from_pretrained("distilgpt2")
     >>> model.config.pad_token_id = model.config.eos_token_id
-    >>> model.generation_config.pad_token_id = model.config.eos_token_id
-    >>> input_context = "Hugging Face Company is"
-    >>> input_ids = tokenizer.encode(input_context, return_tensors="pt")
+    >>> inputs = tokenizer(["Hugging Face Company is"], return_tensors="pt")
 
-    >>> # Without `eos_token_id`, it will generate the default length, 20, ignoring `min_new_tokens`
-    >>> outputs = model.generate(input_ids=input_ids, min_new_tokens=30)
+    >>> # If the maximum length (default = 20) is smaller than the minimum length constraint, the latter is ignored!
+    >>> outputs = model.generate(**inputs, min_new_tokens=30)
     >>> print(tokenizer.decode(outputs[0], skip_special_tokens=True))
     Hugging Face Company is a company that has been working on a new product for the past year.
 
-    >>> # If `eos_token_id` is set to ` company` it will take into account how many `min_new_tokens` have been generated
-    >>> # before stopping. Note that ` Company` (5834) and ` company` (1664) are not actually the same token, and even
-    >>> # if they were ` Company` would be ignored by `min_new_tokens` as it excludes the prompt.
-    >>> outputs = model.generate(input_ids=input_ids, min_new_tokens=1, eos_token_id=1664)
+    >>> # For testing purposes, let's set `eos_token_id` to "company", the first generated token. This will make
```

Comment:
```suggestion
    >>> # For testing purposes, let's set `eos_token` to `"company"`, the first generated token. This will make
```
Let's be more specific hehe